src/toil_lib/programs.py

```diff
@@ -121,6 +153,14 @@ def _fix_permissions(base_docker_call, tool, work_dir):
     :param str work_dir: Path of work directory to recursively chown
     """
     base_docker_call.append('--entrypoint=chown')
+    # We don't need the cleanup container to persist.
+    base_docker_call.append('--rm=True')
     stat = os.stat(work_dir)
     command = base_docker_call + [tool] + ['-R', '{}:{}'.format(stat.st_uid, stat.st_gid), '/data']
     subprocess.check_call(command)
+
+
+def _get_container_name(job):
+    return '--'.join([job.fileStore.jobStore.config.workflowID,
+                      job.fileStore.jobID,
+                      str(uuid.uuid4())[:10]])
```

Comment:
oops it was supposed to be `uuid.uuid4().hex`. Still want me to use pseudo-random strings?